packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -101,6 +101,10 @@ class DraggableScrollableController extends ChangeNotifier {
   /// The duration must not be zero. To jump to a particular value without an
   /// animation, use [jumpTo].
   ///
+  /// The sheet will not snap after calling [animateTo] even if it is not at a
+  /// snap point and [DraggableScrollableSheet.snap] is true. Snapping only
+  /// occurs after user drags.
```

Comment:
I think the `even if it not as a snap point` is redundant and can be removed. Maybe just

> The sheet will not snap after calling animateTo even if [DraggableScrollSheet.snap] is true. Snapping only occurs after user drags.

We should also document this on `DraggableScrollableSheet.snap` and maybe on the top level docs, e.g.

> The snap property only affects user drags. Programmatically manipulating the scroll offset via [DraggableScrollableController.animteTo] or [DraggableScrollableController.jumpTo] will ignore the [snap] property and the snap offsets.

Reply:
This all seems reasonable. Did the following:
1) Removed `even if it is not at a snap point`
2) Added `above snapping clarification (with minor edits) to class level docs
3) Added more brief version of snaping clarification to `snap` attribute docs